akka-remote/src/main/scala/akka/remote/RemoteActorRefProvider.scala

```diff
@@ -389,7 +395,20 @@ private[akka] class RemoteActorRefProvider(
     }
   }
 
-  def resolveActorRef(path: String): ActorRef = path match {
+  def resolveActorRef(path: String): ActorRef = {
+    // using thread local LRU cache, which will call internalRresolveActorRef
+    // if the value is not cached
+    actorRefResolveThreadLocalCache match {
+      case null ⇒ internalResolveActorRef(path) // not initalized yet
```

Comment:
better safe than sorry here, in case resolveActroRef is called for some reason before init of the provider